.github/workflows/axosyslog-docker.yml

```diff
@@ -149,13 +151,21 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Extract metadata (AxoSyslog version) for Docker
-        id: docker-metadata-tags
-        uses: docker/metadata-action@v5
-        with:
-          images: ${{ needs.prepare.outputs.image-name }}
-          tags: type=match,pattern=axosyslog-([0-9].*),group=1
-          sep-tags: ','
+      - id: docker-metadata-tags
+        name: Extract metadata (AxoSyslog version) for Docker
+        run: |
+          last_full_version=$(skopeo inspect docker://ghcr.io/axoflow/axosyslog:latest | jq '.RepoTags' | sed 's/ \|"\|,//g' | grep "^[0-9]*.[0-9]*.[0-9]*$\|^[0-9]*.[0-9]*.[0-9]*-[0-9]*$" | tail -n 1)
```

Comment:
The summary of the patch contains everything twice.

Reply:
Thanks for the review notice, fixed.